exercises/practice/game-of-life/.meta/proof.ci.wat

```diff
@@ -0,0 +1,47 @@
+(module
+  (memory (export "mem") 1)
+
+  ;;
+  ;; Create the next frame of Conway's Game of Life
+  ;;
+  ;; @param {i32} offset - The offset of the current frame in linear memory
+  ;; @param {i32} cols - The number of columns in the frame
+  ;; @param {i32} rows - The number of rows in the frame
+  ;;
+  ;; @returns {(i32,i32)} - The offset and length of the next frame in linear memory
+  ;;
+  (func (export "next") (param $offset i32) (param $cols i32) (param $rows i32) (result i32)
+    (local $pos i32) (local $nbpos i32) (local $length i32) (local $count i32) (local $edges i32) (local $bottom i32)
```

Comment:
Have you looked at the style used in the other `.meta/proof.ci.wat` files?

We usually have one local defined per line.


Comments and indentation can make an example solution easier to follow  - see  [minesweeper](https://github.com/exercism/wasm/blob/main/exercises/practice/minesweeper/.meta/proof.ci.wat)